test/helpers/test-server.js

```diff
@@ -8,6 +8,14 @@ let server;
 // start server, returning the full setup of the server
 // (both the server and the compiler)
 function startFullSetup(config, options, done) {
+  if (typeof options.static === 'undefined') {
+    options.static = {
+      watch: {
+        poll: true,
+      },
+    };
+  }
```

Comment:
It still will create problems with to o many files opens when you try to tests the `watch` option